docs/recipes/build/remove-fake.md

```diff
@@ -0,0 +1,43 @@
+# How do I remove the use of FAKE?
+[FAKE](https://fake.build/) is a tool for build automation. The standard SAFE template comes with a [ready-made build project](/template-safe-commands) at the root of the solution that provides support for many common SAFE tasks.
+
+If you would prefer not to use FAKE, you can of course simply ignore it, but this recipes shows how to completely remove it from your repository. It is important to note that having removed FAKE, you will have to follow a more manual approach to each of these processes. This recipe will only include instructions on how to build and deploy the application after removing FAKE.
+
+> Note that the minimal template does not use FAKE by default, and **this recipe only applies to the standard template**.
+
+#### 1. Build project
+Delete `Build.fs`, `Build.fsproj`, `Helpers.fs`, `paket.references` at the root of the solution.
+#### 2. Dependencies
+Find the following dependencies inside the `paket.dependencies` file that’s also at the root of the solution and delete them.
+Remove the following dependencies 
+```fsharp
+dotnet paket remove Fake.Core.Target
+dotnet paket remove Fake.IO.FileSystem
+dotnet paket remove Farmer
+```
+
+## Running the App
+Now that you have the FAKE dependencies removed, you will have to separately run the server and the client.
+#### 1. Start the Client
```

Comment:
Might it be better to start the server first and then the client? That way an "on initialisation" calls will work.